lib/modules/dosomething/dosomething_campaign/includes/CampaignTransformer.php

```diff
@@ -40,6 +40,7 @@ public function index($parameters) {
     }
     catch (Exception $error) {
       // @TODO: Potentially log error to watchdog.
+      http_response_code('404');
```

Comment:
This was totes a mistake on my part when we were talking things through this morning. This should still return `200` with an empty data array if there are no matching results.